Code/Framework/AzCore/AzCore/Memory/SimpleSchemaAllocator.h

```diff
@@ -53,7 +52,14 @@ namespace AZ
 
         AllocatorDebugConfig GetDebugConfig() override
         {
-            return AllocatorDebugConfig();
+            return {
+#if defined(__cpp_designated_initializers)
+                .m_excludeFromDebugging = !ProfileAllocations,
+#else
+                /* .m_stackRecordLevels = */ 0,
+                /* .m_excludeFromDebugging = */ !ProfileAllocations,
+#endif
```

Comment:
question: Is it really worth adding this define now without us supporting C++20 yet? I agree they're cool and it would be great if we could use them but in the vast majority of cases we cannot

Reply:
I just want to say that I want to set the `m_excludeFromDebugging` member differently from its default value. In C++17 I have no way to express that. With this, when compiled with C++20, it will get the default value for `m_stackRecordLevels` from the member's declaration.

Reply:
You could just set after it is assigned to a variable...